src/components/EditorCommon/config/Rules.js

```diff
@@ -1,5 +1,13 @@
+import { OpenapiDocs } from 'autocomplete-openapi/src/request-docs';
+
 const Method = ['POST', 'GET', 'PUT', 'DELETE', 'PATCH', 'HEAD'];
 
+const response = await fetch(import.meta.env.BASE_URL + './openapi.json');
+const openapi = await response.json();
```

Comment:
```suggestion
import { OpenapiDocs } from 'autocomplete-openapi/src/request-docs';
import { readFile } from 'fs/promises';

const Method = ['POST', 'GET', 'PUT', 'DELETE', 'PATCH', 'HEAD'];

let openapi;
if (import.meta.env.MODE === 'test') {
  const response = await readFile('public/openapi.json', 'utf8');
  openapi = JSON.parse(response);
} else {
  const response = await fetch(import.meta.env.BASE_URL + 'openapi.json');
  openapi = await response.json();
}
```
@Snehil-Shah, When running tests, vitest does not serve local files, resulting in the error you encountered.